wiki/BBCode/fr.md

```diff
@@ -1,395 +1,304 @@
+La balise `[strike]` est utilisée pour indiquer la suppression d'un texte précédemment inclus par l'utilisation d'une ligne horizontale qui "barre" le texte (c'est-à-dire un "barré").
 
-- Bouton dédié : ![Bouton pour rayer du texte](img/strike.png)
-- Notes :
-  - A utiliser avec parcimonie. Une utilisation abusive peut être incomfortable et rendre le sujet plus difficile a lire.
-- Syntaxe :
-  ```
-  [strike] ... [/strike]
-  ```
+Bouton de la barre d'outils : ![Strike button](img/strike.png "Barré")
 
 ### Couleur
 
-*Le BBCode appelle cela "Couleur".*
-
-**Ajoute des couleurs** au texte.
-
-- Bouton dédié : Non assigné
-- Notes :
-  - La boite de couleur est sur le côté droit de la zone de texte.
-  - A utiliser avec parcimonie. Une utilisation abusive peut être incomfortable pour les yeux des lecteurs. Une mauvaise utilisation rendra votre texte illisible.
-  - Quand vous choississez une couleur, soyez sûr qu'il soit lisible par rapport à la couleur du fond.
-  - L'argument `HEXCODE` ne doit pas être entouré de guillemets.
-- Syntaxe :
-  - Où `HEXCODE` est un nom de couleur ou son code hexadécimal  Where `HEXCODE` is a colour in hexadecimal or colour name.
-    - Pour le code hexadécimal, il doit commencer par `#` suivit par 6 caractères hexadécimal (0 - 9, A - F).
-    - Pour le nom de couleur, il doit s'agir d'une couleur HTML valide.
-  ```
-  [color=HEXCODE] ... [/color]
-  ```
-
-Pour une liste du nom des couleurs, voyez [noms de couleur X11](https://fr.wikipedia.org/wiki/Noms_de_couleur_X11#Noms_de_couleur_identiques_entre_X11_et_HTML/CSS).
-
-### Taille de police
-
-Ajuste relativement la taille du texte.
-
-- Bouton dédié : ![option de taille de police](img/font-size.png)
-- Notes :
-  - Utilisez cette balise avec parcimonie.
-  - Vous pouvez utiliser uniquement 4 tailles :
-    - `50` (Très petite)
-    - `85` (petite)
-    - `100` (normale; par défaut)
-    - `150` (grande)
-  - Si vous entrez une valeur qui n'est pas une de celles listées ci-dessus, la valeur par défaut sera utilisée.
-  - L'argument `NUMBER` ne doit pas être entourés de guillemets.
-- Syntax :
-  - Où `NUMBER` est une des tailles suivantes (`50`, `85`, `100`, ou `150`) Where `NUMBER` is one of the listed sizes (`50`, `85`, `100`, or `150`).
-  ```
-  [size=NUMBER] ... [/size]
-  ```
+```
+[color=HEXCODE]texte[/color]
+```
+
+*Pour une liste de tous les noms de couleurs, voir [X11 color names](https://en.wikipedia.org/wiki/X11_color_names#Color_name_chart "Wikipédia (en anglais)")*
+
+La balise `[color]` est utilisée pour styliser le texte au moyen de différents types de couleurs sûres pour le web. Cette balise utilise le format [HEX code](https://en.wikipedia.org/wiki/Web_colors#Hex_triplet "Wikipedia (en anglais)") pour spécifier la couleur, bien qu'elle puisse aussi être spécifiée par des noms de couleur HTML comme "rouge" ou "vert". Pour spécifier, remplacez l'argument `HEXCODE` par le code HEX ou le nom HTML correspondant à la couleur.
```

Comment:
```suggestion
La balise `[color]` est utilisée pour styliser le texte au moyen de différents types de couleurs sûres pour le web. Cette balise utilise des valeurs [RGB héxadécimales](https://fr.wikipedia.org/wiki/Couleur_du_Web#Codage_informatique_des_couleurs "Wikipedia") pour spécifier la couleur, bien qu'elle puisse aussi être spécifiée par des noms de couleur HTML comme "rouge" ou "vert". Pour spécifier, remplacez l'argument `HEXCODE` par le code HEX ou le nom HTML correspondant à la couleur.
```
Unsure about this change I'm suggesting, it's just that I'm not a fan of using "format HEX code".